ui/pages/Staff/constants.js

```diff
@@ -13,7 +13,10 @@ import {
   INDIVIDUAL_FIELD_MATERNAL_ID,
   INDIVIDUAL_FIELD_SEX,
   INDIVIDUAL_FIELD_AFFECTED,
+  FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP,
 } from 'shared/utils/constants'
+import {ColoredIcon} from "../../shared/components/StyledComponents";
```

Comment:
this is definitely failing the linter. Before pushing up code, please remember to always check and make sure you don't see any errors